demo/reset-counters.sh

```diff
@@ -7,9 +7,10 @@ source .env
 
 # This script assumes that port forwarding has already been established.
 # See ./scripts/port-forward-all.sh to enable fort forwarding for Bookstore demo.
-# Requires GNU Parallel
-parallel curl -I -X GET ::: \
-         http://localhost:8080/reset \
-         http://localhost:8081/reset \
-         http://localhost:8082/reset \
-         http://localhost:8083/reset
+curl -I -X GET http://localhost:8080/reset &
```

Comment:
Should we add some arbitrary timeout (`--connect-timeout`)? Can get stuck for some time otherwise (order of minutes from what I'm [reading](https://unix.stackexchange.com/questions/94604/does-curl-have-a-timeout))

Reply:
Would make it a separate change if the problem exists prior to this change.